app/src/main/java/com/example/triptracker/viewmodel/UserProfileViewModel.kt

```diff
@@ -24,8 +25,15 @@ import kotlinx.coroutines.launch
 class UserProfileViewModel(
```

Comment:
We could add comments for the parameter of the view model (even though they have default values)